### PR TITLE
VIH-10899 run chrome headless

### PR DESCRIPTION
**Pull request by shaed-parkar**
### Change description

Run chrome headless if on
Produce a single page html report for UI test results

### CODE REVIEW

File: UI.AutomationTests/Admin/Booking/EditMultiDayHearingTests.cs [hunk 1]
```diff
@@ -131,11 +131,14 @@ private SummaryPage EditMultiDayHearing(BookingDto hearingDto, SummaryPage summa
             
             // Go to the summary page
             // The journey is slightly different here when special measures is enabled
-            var specialMeasuresEnabled = FeatureToggle.Instance().SpecialMeasuresEnabled();
-            if (specialMeasuresEnabled)
-                summaryPage = videoAccessPointsPage.GoToSummaryPage();
-            else
-                summaryPage = videoAccessPointsPage.GoToOtherInformationPage().GoToSummaryPage();
+            // TODO: confirm steps with a dev
```
Comment (will-craig) [inline UI.AutomationTests/Admin/Booking/EditMultiDayHearingTests.cs:134]: Does this TODO need to be done before merging?

File: UI.AutomationTests/Admin/Booking/EditMultiDayHearingTests.cs [hunk 2]
```diff
@@ -131,11 +131,14 @@ private SummaryPage EditMultiDayHearing(BookingDto hearingDto, SummaryPage summa
...
+            summaryPage = videoAccessPointsPage.GoToSummaryPage();
+            
+            // var specialMeasuresEnabled = FeatureToggle.Instance().SpecialMeasuresEnabled();
+            // if (specialMeasuresEnabled)
+            //     summaryPage = videoAccessPointsPage.GoToSummaryPage();
+            // else
+            //     summaryPage = videoAccessPointsPage.GoToOtherInformationPage().GoToSummaryPage();
```
Comment (will-craig) [inline UI.AutomationTests/Admin/Booking/EditMultiDayHearingTests.cs:141]: This linked to https://tools.hmcts.net/jira/browse/VIH-11118 ?

File: UI.AutomationTests/UI.AutomationTests.csproj [hunk 3]
```diff
@@ -8,8 +8,9 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="BookingsApi.Client" Version="3.0.5" />
-        <PackageReference Include="ExtentReports" Version="4.1.0" />
+        <PackageReference Include="BookingsApi.Client" Version="3.1.2-pr-0919-0015" />
```
Comment (will-craig) [inline UI.AutomationTests/UI.AutomationTests.csproj:11]: Need to update to latest package

File: UI.AutomationTests/Admin/Booking/BookHearingTests.cs [hunk 4]
```diff
@@ -6,23 +6,24 @@ public class BookHearingTests : AdminWebUiTest
...
         var date = DateTime.Today.AddDays(1).AddHours(10).AddMinutes(30);
         _bookingDto = HearingTestData.CreateHearingDtoWithEndpoints(HearingTestData.JudgePersonalCode,
             judgeUsername: HearingTestData.JudgeUsername, scheduledDateTime: date);
         _bookingDto.CaseNumber = $"Automation Test Hearing - BookAHearing {Guid.NewGuid():N}";
-        var newUser = HearingTestData.CreateNewParticipantDto();
-        CreatedUsers.Add(newUser.Username);
-        _bookingDto.NewParticipants.Add(newUser);
+        // var newUser = HearingTestData.CreateNewParticipantDto();
+        // CreatedUsers.Add(newUser.Username);
+        // _bookingDto.NewParticipants.Add(newUser);
```
[the remark this answers is not in the document]
Comment (shaed-parkar) [reply UI.AutomationTests/Admin/Booking/BookHearingTests.cs:20]: thanks needed to revert

File: UI.AutomationTests/Admin/Booking/EditMultiDayHearingTests.cs (hunk 2 above)
Comment (shaed-parkar) [reply UI.AutomationTests/Admin/Booking/EditMultiDayHearingTests.cs:141]: reverting

File: UI.AutomationTests/UI.AutomationTests.csproj (hunk 3 above)
Comment (shaed-parkar) [reply UI.AutomationTests/UI.AutomationTests.csproj]: thanks, QA should hopefully wrap up soon

**Review by komaldabhi1509 — APPROVED** [review]

**Review by will-craig — APPROVED** [review]

**Pull request merged**